packages/chat-component/src/main.ts

```diff
@@ -252,12 +258,6 @@ export class ChatComponent extends LitElement {
     .user-message .chat__txt--info {
       text-align: right;
     }
-    .items__list {
```

Comment:
Why is items__list deleted? It styled all <uls>

Reply:
because we now use <ol> which add the list item numbers. this css rule was conflicting.